H2MLauncher.Core/Services/H2MCommunicationService.cs

```diff
@@ -83,7 +83,50 @@ internal static IEnumerable<IntPtr> EnumerateWindowHandles()
 
             return null;
         }
-        
+
+        private (string fileName, bool found) TryFindValidGameFile()
```

Comment:
use `TryFindValidGameFile(out string fileName)` and return bool

Reply:
The reason i did Not is because i always return the attempted file Name. Is it still preferred in this case?